.eslintrc.json

```diff
@@ -82,10 +85,10 @@
         "src/graphVisualization/**/*.tsx"
       ],
       "rules": {
-        "react/prefer-stateless-function": [
-          "error",
-          { "ignorePureComponents": true }
-        ],
+        //        "react/prefer-stateless-function": [
```

Comment:
Let's just remove this completely, no need to have the comment laying around!

Reply:
Sure I was leaving it there since I was planning to take it back in later PR, but it is better I remove it and just take it back then.